spec/models/taxonomy/expanded_taxonomy_spec.rb

```diff
@@ -77,7 +119,7 @@ def home_page
 
   describe 'Ask for the Homepage' do
     before :each do
-      @taxonomy = ExpandedTaxonomy.new(GovukTaxonomy::ROOT_CONTENT_ID).build
+      @taxonomy = Taxonomy::ExpandedTaxonomy.new(GovukTaxonomy::ROOT_CONTENT_ID).build
```

Comment:
I'm not sure why this is changing?

Reply:
I believe these changes are necessary because the code used to be enclosed in a module statement and now uses RSpec.describe Taxonomy::ExpandedTaxonomy to satisfy the Linters maximum module length.

I'll put these changes in a separate commit

Reply:
Awesome, thanks 👍